package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ripple-lib-extensions",
```

Comment:
not necessarily - I left all the packages the same, only added a higher level one for gulp-webpack to compile all at the same time. But I could just as easily set it up so that each package has a gulpfile.